tests/multivariate/test_tree.py

```diff
@@ -0,0 +1,42 @@
+from unittest import TestCase
+
+import numpy as np
+import pandas as pd
+
+from copulas.multivariate.Tree import DTree, Edge
+from copulas.univariate.KDEUnivariate import KDEUnivariate
+
+
+class TestTree(TestCase):
```

Comment:
Unittests should be created for all methods of the `Tree` sublasses.Each subclass under its own `TestCase`.
Also, docstrings explaining the expected/tested behavior should be added to each method, as the test runner used it as error message when the test fails.